### ISSUE TITLE
00 -- Jeremy Rist

### ISSUE THREAD

**Issue by jeremyrist** [TIY-Durham/2015-FALL-FEE#11]
- [x] **GitHub-ing to Know You**
  - [x] [Create a GitHub account](https://github.com/signup)
  - [x] [Edit your GitHub profile](https://github.com/settings/profile) to provide:
  - [x] A picture, so we can recognize your face.
  - [x] Your full name, so we can recognize your name.
  - [x] A valid public email address, so we can contact you.
  - [x] Find and star [the class project](https://github.com/TIY-Durham/2015-FALL-FEE)
  - [x] Read [the Home page on the class wiki](https://github.com/TIY-Durham/2015-FALL-FEE/wiki)
- [x] **You've Got Issues**
  - [x] [Create a new _WIP Issue_](https://github.com/TIY-Durham/2015-FALL-FEE/issues/new)
  - [x] Name it `00 -- YOUR NAME` (where `YOUR NAME` is _your_ name)
  - [x] Make this checklist the description
  - [x] Start checking off boxes... :astonished:
- [ ] **Send all my GitHub to you...**
  - [ ] Learn all you can about `git` and GitHub!
    - Treehouse has [a Git Basics course](http://teamtreehouse.com/library/git-basics)
  - CodeSchool has [a `git` path](https://www.codeschool.com/paths/git)
    - [x] Complete: [Try Git](http://try.github.com)
  - [x] GitHub has a couple of Guides:
    - [x] [Hello World](https://guides.github.com/activities/hello-world/)
    - [x] [Forking Projects](https://guides.github.com/activities/forking/)
  - [x] Add a link to your `Hello-World` and `Fork-and-Spoon` repos as a comment on your _WIP Issue_
- [ ] **Command Line Training**
  - [x] Read: http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/
  - [ ] Complete: https://www.codecademy.com/courses/learn-the-command-line/
  - [ ] **BEAST MODE** http://cli.learncodethehardway.org/book/
- [x] **INSTALL ALL THE THINGS!**
  - [x] [Install Homebrew](http://brew.sh) on your Mac
  - [x] Open `Terminal.app`; Don't Panic.
  - [x] [Install Homebrew and the `brew` command](http://brew.sh/#install)
  - [x] Read [the Homebrew Wiki](https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme) to learn more about Homebrew
  - [x] Using the `brew` command, install the following packages:
    - [x] `zsh`
  - [x] `git`
  - [x] `hub`
  - [x] `node`
  - [x] Add the output of the following commands as comments on your _WIP Issue_:
    - [x] `which zsh && zsh --version`
    - [x] `which git && git --version`
    - [x] `which hub && hub --version`
    - [x] `which node && node --version`
  - [x] Install the following (bonus if you use `brew`):
    - [x] [Google Chrome](http://google.com/chrome)
    - [x] [Atom](http://atom.io)
    - [x] [Dash](http://kapeli.com/dash)
    - [x] [Slack](http://www.slack.com)
    - [x] [SourceTree](https://www.sourcetreeapp.com/)
- [x] **Prezto Change-oh!**
  - [x] Find and _fork_ the Prezto project on GitHub
  - [x] Clone _your fork_ of Prezto into your home directory
  - [x] Follow the installation instructions in the `README.md` file
- [x] **Take a Moment to Reflect**
  - [x] Answered question 1 in a comment on your WIP Issue.
  - [x] Answered question 2 in a comment on your WIP Issue.

**Comment by jeremyrist** [comment]
https://github.com/J-Rist/hello-world
https://github.com/J-Rist/Spoon-Knife

**Comment by J-Rist** [comment]
which zsh && zsh --version

**Comment by jeremyrist** [comment]
/usr/local/bin/zsh
zsh 5.0.8 (x86_64-apple-darwin14.3.0)

**Comment by jeremyrist** [comment]
/usr/local/bin/git
git version 2.4.6

**Comment by jeremyrist** [comment]
/usr/local/bin/hub
git version 2.4.6
hub version 2.2.1

**Comment by jeremyrist** [comment]
/usr/local/bin/node
v0.12.7

**Comment by jeremyrist** [comment]
Difficulties I expect for the class are about what I would expect from any new class. I was a literature and history guy in college so most of my classes involved a lot of reading and interpretation. I had more trouble in language and economics classes and that seems to be what a lot of this coding stuff reminds me of, sort a mix of the two. It's about learning new concepts and ways of communicating but still using English phrases (just in a new context). In the past I've simply tried my best to master the concepts and move forward and that's what I plan to do here. I'll ask questions when I have them and otherwise try to work things out on my own.

**Comment by jeremyrist** [comment]
I enjoyed working in codecademy doing the html and css stuff. I also enjoyed the command line training. Basically I liked everything that came easy to me and involved copying and pasting and the more difficult parts of javascript got me frustrated. I'm also having a little trouble keeping github, Git, and Terminal all straightened out. It's a very new thing for me to jump back forth between talking directly to my computer, then to a web page, and then back to the computer. But I'm sure I'll get used to it!
